lib/components/textfield/text_field.dart

```diff
@@ -102,68 +101,67 @@ class TUIInputField extends StatelessWidget {
-            enabled: enabled,
-            obscureText: obscureText,
-            obscuringCharacter: obscuringCharacter,
-            textAlign: textAlign,
-            textAlignVertical: textAlignVertical,
-            textCapitalization: textCapitalization,
-            textDirection: textDirection,
-            minLines: minLines,
-            maxLines: maxLines,
-            maxLength: maxLength,
-            decoration: InputDecoration(
-              filled: true,
-              fillColor: TUIDefaultColors.inputBackground,
-              hintText: hintText,
-              hintStyle: TUITextStyle.body6
-                  .copyWith(color: TUIDefaultColors.inputTextDim),
-              labelText: labelText,
-              labelStyle: TUITextStyle.body6
-                  .copyWith(color: TUIDefaultColors.inputTextDim),
-              suffixText: suffixText,
-              prefixText: prefixText,
-              prefixIconColor: prefixIconColor,
-              prefixIcon: prefixIcon ?? prefixIcon,
-              suffixIconColor: TUIDefaultColors.inputText,
-              suffixIcon: suffixIcon ?? suffixIcon,
-              errorText: errorText,
-              helperText: helperText,
-              errorStyle: TUITextStyle.body7
-                  .copyWith(color: TUIDefaultColors.inputText),
-              helperStyle: TUITextStyle.body7
-                  .copyWith(color: TUIDefaultColors.inputText),
-              focusedBorder: const UnderlineInputBorder(
-                  borderRadius: BorderRadius.all(Radius.circular(8)),
-                  borderSide: BorderSide(color: TUIDefaultColors.primary)),
-              disabledBorder: const UnderlineInputBorder(
-                  borderRadius: BorderRadius.all(Radius.circular(8)),
-                  borderSide:
-                      BorderSide(color: TUIDefaultColors.inputBackground)),
-              enabledBorder: const UnderlineInputBorder(
-                  borderRadius: BorderRadius.all(Radius.circular(8)),
-                  borderSide:
-                      BorderSide(color: TUIDefaultColors.inputBackground)),
-              border: const UnderlineInputBorder(
-                  borderRadius: BorderRadius.all(Radius.circular(8)),
-                  borderSide:
-                      BorderSide(color: TUIDefaultColors.inputBackground)),
-              errorBorder: const UnderlineInputBorder(
-                  borderRadius: BorderRadius.all(Radius.circular(8)),
-                  borderSide: BorderSide(color: TUIDefaultColors.error)),
-              focusedErrorBorder: const UnderlineInputBorder(
-                  borderRadius: BorderRadius.all(Radius.circular(8)),
-                  borderSide: BorderSide(color: TUIDefaultColors.error)),
-            )));
+    final theme = TUITheme.of(context);
+    final suffixIconColor = this.suffixIconColor ?? theme.colors.inputText;
+    final prefixIconColor = this.prefixIconColor ?? theme.colors.inputText;
+    return TextField(
```

Comment:
@kalpeshp0310 In the previous version the Textfield was a child of Material, now it is removed. I am curious as to why we have removed it now.

Reply:
I am not sure why it was wrapped inside `Material` in the first place. I do not see any value in doing that.

Reply:
Material widget enhances the visual appeal and interactivity for example it provides clipping, elevation, and ink effects for UI elements.
https://api.flutter.dev/flutter/material/Material-class.html

Reply:
I feel `Material` should only be used when applying material effects on non-material components. Since TextField is already a material component and adheres to all the material standards like shadow, elevation, ink effect, etc.

Reply:
Got you but I am not sure about this, can you please share some documentation link / blog so that I can read further about it.

Reply:
Not the most reliable source but ai suggests that it can be helpful.
https://share.zight.com/yAuBwzyK